/wbc/joint_limits/controller:
  ros__parameters:
    control_rate: 250.0
    influence_distance: [0.1,0.1,0.1,0.1,0.1,0.1,0.1]
    p_gain: [1.0,1.0,1.0,1.0,1.0,1.0,1.0]
    max_control_output: [1.0,1.0,1.0,1.0,1.0,1.0,1.0]
    joint_limits:
        names: ["joint_0","joint_1","joint_2","joint_3","joint_4","joint_5","joint_6"]
        joint_0:
          position:
              max: 2.96
              min: -2.96
        joint_1:
          position:
              max: 2.09
              min: -2.09
        joint_2:
          position:
              max: 2.96
              min: -2.96
        joint_3:
          position:
              max: 2.09
              min: -2.09
        joint_4:
          position:
              max: 2.96
              min: -2.96
        joint_5:
          position:
              max: 2.09
              min: -2.09
        joint_6:
          position:
              max: 3.05
              min: -3.05
